src/main/java/eu/bbmri_eric/negotiator/exceptions/NegotiatorExceptionHandler.java

```diff
@@ -275,4 +279,22 @@ public final ResponseEntity<HttpErrorResponseModel> handleDataIntegrityViolation
             .build();
     return new ResponseEntity<>(errorResponse, HttpStatus.BAD_REQUEST);
   }
+
+  @ExceptionHandler(NullPointerException.class)
```

Comment:
I think this is too risky. NullPointerException may be a programming error and you usually don't have control over it. You would end up here each time an uncontrolled NullPointerException is thrown. If you have some point where you know it may occur then I suggest you catch it and throw another exception that you can handle here